### DISCUSSION TITLE
Reaction force calculation in OpenRadioss

### DISCUSSION THREAD

**Discussion by Majid-MHZ** [OpenRadioss/OpenRadioss#2773] [Q&A] [answered]
Dear @MarianBulla,
I hope you are doing well.

Thank you for your support. I watched your YouTube video titled "How to Postprocess OpenRadioss Results - Part 2," which covers postprocessing simulations with Paraview software. In the video, you utilized data from a .csv file to calculate the reaction force. I have a couple of questions about this process. First, why did you use the first derivative to calculate the reaction force? What are the numbers that were written in the column labeled 'TH-RBODY'?

Best regards,
Majid

**Comment by MarianBulla** [comment, marked as answer]
Dear Majid,

first, thanks for your deeper interest in OpenRadioss.
In OpenRadioss, in the timehistory file ( T01, T02, T0x...) we do not store any forces. We store the impulse.  I = m * v
If you do the derivative by time, you will obtain the force. 
This is, what is done automatically in the postprocessors like HyperView.
But if you extract the data, using the small tool, to a cvs file, you have to do it manually, as I have shown in the YouTube video.

This is different for energy, which is stored as it is, as energy.

Hope it helps you ?

Thanks and best regards,
Marian

**Comment by Majid-MHZ** [comment]
Dear @MarianBulla 

I truly appreciate your thorough explanation.

Best wishes,
Majid
